voicebox/__main__.py

```diff
@@ -10,25 +10,48 @@
     format='%(asctime)s %(name)s %(levelname)-8s  %(message)s',
     datefmt='(%H:%M:%S)'
 )
-
-
-def main():
-
-    """
-    Main function to start system
-
-    provides a command line interface for interracting
-    with the voicebox APIs.
-    """
-
+server = Server()
+async def run(port, bootstrap_ip=None, bootstrap_port=None):
```

Comment:
Consider adding error handling for the `run` function. If the server fails to listen on the specified port or bootstrap with the provided node, the function will return False, but the main function will continue to execute. This could lead to unexpected behavior. [important]